docs/source/virtual_rainforest/soil/soil_details.md

```diff
@@ -111,30 +111,30 @@ significant nitrogen cycle processes that involve inorganic forms of nitrogen. F
 reason a number of inorganic nitrogen pools are additionally defined. They are as
 follows:
 
-### Combined ammonia ($\\ce{NH\_{3}}$) and ammonium ($\\ce{NH\_{4}^{+}}$) pool
+### Combined ammonia ($\ce{NH_{3}}$) and ammonium ($\ce{NH_{4}^{+}}$) pool
```

Comment:
Are these rendering right now? The [read the docs site](https://virtual-rainforest.readthedocs.io/en/feature-getting_docs_working/virtual_rainforest/soil/soil_details.html) still shows them as badly formatted, but I can't tell if the site is showing a previous version

Reply:
They look great, now! You can see the revision rendered at the bottom of the page.